pages/k8s/charm-kubernetes-worker.md

```diff
@@ -352,21 +352,307 @@ update the configuration for the workers:
     ```
 
 ## Actions
-
-The kubernetes-worker charm models a few one time operations called
-[Juju actions](https://jaas.ai/docs/working-with-actions) that can be run by
-Juju users.
-
-### Pause
-
-Pausing the workload enables administrators to both drain and cordon
-a unit for maintenance.
-
-### Resume
-
-Resuming the workload will uncordon a paused unit. Workloads will automatically
-migrate unless otherwise directed via their application declaration.
-
+<!-- ACTIONS STARTS -->
+<!-- AUTOGENERATED TEXT - DO NOT EDIT -->
+
+The following actions are available. You can run an action with the following
+
+The following actions are available. You can run an action with the following ```bash juju run-action kubernetes-worker ACTION [parameters] [--wait] ```
+<div class="row">
+  <div class="col-2">
+    <h5>
+      cis-benchmark
+    </h5>
+  </div>
+  <div class="col-7">
+    <p>
+      Run the CIS Kubernetes Benchmark against snap-based components.
+    </p>
+  </div>
+</div>
+<div class="row">
+  <div class="col-2"></div>
+  <div class="col-7">
+    <p>
+      This action has the following parameters:
+    </p>
+    <table class="u-table-layout--auto">
+      <thead>
+        <tr>
+          <th>
+            parameter
+          </th>
+          <th>
+            Description
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <pre>apply</pre>
+          </td>
+          <td>
+            Apply remediations to address benchmark failures. The default, 'none', will not attempt to fix any reported failures. Set to 'conservative' to resolve simple failures. Set to 'dangerous' to attempt to resolve all failures. Note: Applying any remediation may result in an unusable cluster. <strong>Default:</strong> none
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <pre>config</pre>
+          </td>
+          <td>
+            Archive containing configuration files to use when running kube-bench. The default value is known to be compatible with snap components. When using a custom URL, append '#=' to verify the archive integrity when downloaded. <strong>Default:</strong> https://github.com/charmed-kubernetes/kube-bench-config/archive/cis-1.5.zip#sha1=cb8e78712ee5bfeab87d0ed7c139a83e88915530
```

Comment:
The `'#<hash_type>=<checksum>'` was sanitized to `'#='` instead of escaped to `'#&lt;hash_type&gt;=&lt;checksum&gt;'`

Reply:
this should now work. I escaped the text, then htmlified everything, then sanitised it, then made it pretty. <https://www.youtube.com/watch?v=yiJ9fy1qSFI>